lib/_constants.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  CLASS_NAMESPACE: 'gmusic-theme',
+  CLASS_NAMESPACE: 'gmusic-theme.material',
```

Comment:
The question we should be asking is, do we even need `.material` in the namespace.  As far as I know / can tell we can kill it completely.